tests/unit/test_bruteforce.cpp

```diff
@@ -119,22 +119,23 @@ TEST_F(BruteForceTest, resizeNAlignIndex) {
         }
```

Comment:
this function should behave the same for single and multi, consider using BruteForceIndex